components/support/locale/src/main/java/mozilla/components/support/locale/LocaleManager.kt

```diff
@@ -48,6 +50,33 @@ object LocaleManager {
         return currentLocale
     }
 
+    /**
+     * Change the current locale to the system defined one.
+     * [getCurrentLocale] will return null
```

Comment:
nit: What do you think about ` "Change the current locale to the system defined one. As a result, [getCurrentLocale] will return null"`?

Reply:
Changed, thanks